### ISSUE TITLE
Add more documentation

### ISSUE THREAD

**Issue by klnusbaum** [pypt/fervor#1]
So I get the general idea as to how fervor works and I really like it. But even with the sample application I'm still now quite sure how it works? Do I just create an entirely new binary and fervor will take care of the installing and diffing and everything for me? Could you provide some more details in the form of more documentation? Maybe write a wiki article walking through how to take an existing qt application and start using fervor with it.

tl;dr I love this idea! Can you tell me more?
